src/components/Contact.js

```diff
@@ -38,8 +38,7 @@ const Contact = observer(({assembly}) => (
           {assembly.translate("contact.discussion_title")}
         </h3>
 
-        {/* TOOD: In the future we may not want the target to be blank */}
-        <Link href="https://asistente.trydiscourse.com/" target="_blank">
+        <Link href="https://tb-discourse.cirg.washington.edu" target="_blank">
```

Comment:
I've been thinking a bit about the humble old `<iframe/>` here...
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

Reply:
Sadly this is not possible -> https://meta.discourse.org/t/support-embedding-discourse-in-an-iframe/50405/6